corehq/apps/hqadmin/views.py

```diff
@@ -777,7 +777,7 @@ def view_data(case_id, indicators):
             'case': CommCareCase.get(case_id),
         }
 
-    if user or user_case:
+    if (user or user_case) and domain.call_center_config.use_fixtures:
```

Comment:
That's true, however I was wondering if it might lead to confusion if fixtures are show here even when `use_fixtures` is false. I've reverted this change and added a caveat to the paragraph on the page instead.